packages/headless/src/features/commerce/facets/facets-state.ts

```diff
@@ -0,0 +1,9 @@
+import {AnyFacetResponse} from '../../facets/generic/interfaces/generic-facet-response';
+
+export interface CommerceFacetsState {
```

Comment:
This is meant to be compatible with the commerce search solution type, when we add it. We'll just need to add a case for handling the action that fulfills a commerce search in the reducer above.